### DISCUSSION TITLE
an issue on my project is a 404?

### DISCUSSION THREAD

**Discussion by rdp** [community/community#88635] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Hello, this issue: https://github.com/rdp/ffmpeg-windows-build-helpers/issues/430 seems AWOL 404?  Any ideas?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by moheedinamdar** [comment, marked as answer]
The issue may have been deleted by the repository maintainer or contributors. Deleted issues are not accessible, and a 404 error would be displayed. Check if there's any notification or comment from the repository maintainers regarding the deletion.
OR
the repository might have been archived, it might affect the visibility of issues. Archived repositories are read-only, and issues may not be accessible. Check the repository settings.

**Reply by rdp** [reply]
Makes sense, maybe the person who created it asked for deletion :)
